Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-slim
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt /app/
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app/
```

Comment:
Consider using a `.dockerignore` file to exclude unnecessary files and directories from being copied into the Docker image. This can help reduce the image size.